bigquery/deploy/k8s_es_deploy/deploy.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
nit Change directory name k8s_es_deploy -> kubernetes-elasticsearch-cluster to match original github repo. Add README saying this is copied from https://github.com/pires/kubernetes-elasticsearch-cluster with some changes

Reply:
Done